core/src/util/test_connection.rs

```diff
@@ -120,6 +120,7 @@ mod test {
     }
 
     #[test]
+    #[ignore = "SQL-2288: need real libmongosqltranslate"]
```

Comment:
[Note] If it is grabbing the mock library now, it will still later on because it is using cfg!(test).
We'll like have to add mock implementations for the different commands too.